src/seedu/addressbook/AddressBook.java

```diff
@@ -391,6 +399,22 @@ private static String getMessageForInvalidCommandInput(String userCommand, Strin
 		return String.format(MESSAGE_INVALID_COMMAND_FORMAT, userCommand, correctUsageInfo);
 	}
 
```

Comment:
Need header comment. editPerson could mean several types of edits. So the comment needs to clarify the functionality here clearly.